invenio_communities/views/ui.py

```diff
@@ -148,4 +182,16 @@ def register_menus():
     # Register context processor
     blueprint.app_context_processor(search_app_context)
 
+    # Template filters
+    @blueprint.app_template_filter()
+    def format_datetime(value, format="medium"):
```

Comment:
Shouldn't be the date/time formatted by current `locale`?

Reply:
You're right, thanks!